build/configs/integreat/index.js

```diff
@@ -1,14 +1,16 @@
 // @flow
 
-import type { AppConfigType } from '../../src/AppConfigType'
-
-import brightTheme from '../../src/modules/theme/constants/theme'
+import integreatTheme, { darkTheme as darkIntegreatTheme } from '../../themes/integreat'
+import type { AppConfigType } from '../AppConfigType'
+import featureFlags from '../featureFlags'
 
 const IntegreatAppConfig: AppConfigType = {
   appTitle: 'Integreat',
   itunesAppId: '1072353915',
-  theme: brightTheme,
+  theme: integreatTheme,
+  darkTheme: darkIntegreatTheme,
   cmsUrl: 'https://cms.integreat-app.de',
+  featureFlags,
```

Comment:
When writing the issue I also thought that `theme` and `appTitle` etc are feature flags. So you could just add here:
```suggestion
  pois: false,
  newsStream: false
```
When talking about build configs you can say: `I set the feature flag pois is the build config malte to true`.

Reply:
As discussed: will keep this, `feature flags` is a `build option`.